dapp-developer-guide/tutorials/upload-your-file-folder-with-ethfs-cli.md

```diff
@@ -15,48 +16,97 @@ For the content to be uploaded, let's assume that
 
 If you have not already done so, you can install `ethfs-cli` using the following command:
 
-```
+```bash
 npm i -g ethfs-cli
 ```
+If you installed it a long time ago, please run the same command to update to the latest version.
 
 ## Step 2: Create a `FlatDirectory` Contract
 
 A [FlatDirectory](https://docs.web3url.io/advanced-topics/flatdirectory) contract serves as a container that needs to be created before uploading files. 
 
-The following command creates a `FlatDirectory` on the default chain (**Ethereum Mainnet**) using the private key `0x112233...`
+### Command Syntax
 
+```bash
+ethfs-cli create -p <private-key> -c <chain-id>
 ```
-$ ethfs-cli create -p 0x112233...
-```
+This command creates a `FlatDirectory` on a specified blockchain. You need to provide a private key and the blockchain's ID. 
 
-If you intend to create it on other chains that support EVM, you need to add the chain ID using the `-c` flag, for example, `11155111` for **Sepolia**.
+If the part `-c <chain-id>` is omitted, **Ethereum Mainnet** will be specified by default.
```

Comment:
(omitted  => Optional) Is this better?

Reply:
omitted means not provide, it is the case of mainnet